src/org/wojcieszko/homework/homework08/SecondValue.java

```diff
@@ -0,0 +1,32 @@
+package org.wojcieszko.homework.homework08;
+
+import org.junit.Test;
+
+public class SecondValue {
+
+    private int[] table;
```

Comment:
Jako jedyni zdecydowaliście się w tym miejscu na użycie tablicy. 
Większość przykładów z sieci to są dwie oddzielne zmienne. Ani tablica, ani kilka zmiennych tu nie pokazują przewagi.
Gdyby zadanie mówiło o "siedemnastej największej", to wtedy tylko tablica.
Tak, że spory plus :1st_place_medal:

Reply:
Jeśli byśmy chcieli siedemnastą to żeby było optymalnie trzeba by wykorzystać rekurencję? Bo else if na siedemnaście klamer brzmi absurdalnie.

Reply:
Rekurencja będzie dziś. Tu by wystarczyła pętla po sortowanej tablicy. Twoja tablica jest właśnie sortowaną tablicą (najmniejszy na początku)